### ISSUE TITLE
google play

### ISSUE THREAD

**Issue by GuncraftaM1P** [prietzem/spacerex#70]
publish files on google play to male it easier for others to download and test
- also save #66 achievements there

**Comment by prietzem** [comment]
Or fdroid could be an option

**Comment by GuncraftaM1P** [comment]
> Or fdroid could be an option

i think fdroid is more for apps and less for games

**Comment by prietzem** [comment]
Nope, it's not only designed for apps. It's just designed for individual devs to publish their work
